codalab/model/bundle_model.py

```diff
@@ -522,6 +524,25 @@ def make_condition(key, field, value):
                     clause = cl_bundle.c.uuid.in_(
                         alias(select([cl_worksheet_item.c.bundle_uuid]).where(condition))
                     )
+            elif key in ('.before', '.after'):
+                target_datetime = parser.isoparse(value)
+
+                subclause = None
+                if key == '.before':
```

Comment:
You mentioned in another file that .before is exclusive and .after is inclusive. Here, it seems like both are inclusive.

Reply:
I think we want both to be inclusive.